armi/materials/molybdenum.py

```diff
@@ -23,7 +23,7 @@ class Molybdenum(SimpleSolid):
     name = "Molybdenum"
 
     def setDefaultMassFracs(self):
-        """Moly mass fractions."""
+        """Moly mass fractions"""
```

Comment:
The period is generally required by docstring checkers.

Reply:
Wait, really?

Which docstring checkers?  I've never seen that. They render fine in our docs.

The period there is grammatically incorrect.  I've always prescribed to the "don't put punctuation in code comments, unless they are complete sentences".  Which... I'm sure is a rule I have applied far and wide in ARMI.

Reply:
fixed